templates/ces_admin/ces-dashboard.html

```diff
@@ -5,36 +5,45 @@
 
 <div class='row'>
     <div class='col-md-10 offset-md-1'>
-        <div class='row mt-4'>
+        <div class='row mt-6'>
             <div class='col-md-12'>
-                <h2>Open sessions</h2>
-                <h4><span class='badge badge-pill badge-primary'>{{ open_sessions }}</span> users have open sessions</h4>
-                <p>These users started a survey in the last 24 hours.</p>
-                {% if open_sessions %}
-                    <div id="open-chart" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
-                {% endif %}
+                <h2>Overview</h2>
+
+                <div class='row'>
+                    <div class='col-md-4 p-5 text-center'>
+                        <h3>{{ open_sessions }}</h3>
+                        <p>Surveys currently in progress (started in the last 24 hours)</p>
+                    </div>
+                    <div class='col-md-4 p-5 text-center'>
+                        <h3>{{ percentage_complete | floatformat }}%</h3>
+                        <p>Percentage of surveys started that were completed</p>
+                    </div>
+                    <div class='col-md-4 p-5 text-center'>
+                        <h3>{{ percentage_canceled | floatformat }}%</h3>
+                        <p>Percentage of surveys started that were canceled</p>
+                    </div>
+                </div>
             </div>
         </div>
-        <div class='row mt-4'>
+        <div class='row mt-6'>
             <div class='col-md-12'>
-                <h2>Canceled sessions</h2>
-                <h4><span class='badge badge-pill badge-primary'>{{ canceled_sessions }}</span> users canceled their session</h4>
-                <p>These users canceled their session, either by instructing the tool to close thier session (e.g. by typing "end") or by abandoning their session after 24 hours. This chart shows the question or resource on which users canceled their sessions.</p>
-                <div id="canceled-chart" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
+                <h2>How did users end their sessions?</h2>
+                <h4>Completed sessions <span class='badge badge-pill badge-primary'>{{ completed_sessions }}</span></h4>
```

Comment:
Let's put the pills first, so someone could skim down the page and see all the numbers.